src/Components/AddTask/AddTask.tsx

```diff
@@ -0,0 +1,175 @@
+    <>
+      <div className="header d-flex justify-content-between p-3 bg-light">
+        <div className="">
+          <button 
+          onClick={goBack}
+          className="btn ">&laquo; View All Tasks</button>
+          <h2>Add New Task</h2>
+        </div>
+        
+      </div>
+      <div className="vh-50 w-75  bg-light shadow-lg rounded-4">
+        <form
+          onSubmit={handleSubmit(onSubmit)}
+          action=""
+          className="form-wrapper m-auto w-75  pt-5 pb-3 px-5"
+        >
+          <div className="form-group my-3">
+            <label className="label-title mb-2">Title</label>
+            <input
+              {...register("title", {
+                required: true,
+              })}
+              type="text"
+              name="title"
+              className="form-control"
+              placeholder="Enter Title..."
+            />
+
+            {errors.title && errors.title.type === "required" && (
+              <span className="text-danger ">title is required</span>
+            )}
+          </div>
+          <div className="form-group my-3">
+            <label className="label-title mb-2">Description</label>
+            <textarea
+              {...register("description", {
+                required: true,
+              })}
+              rows={5}
+              type="text"
+              name="description"
+              className="form-control"
+              placeholder="Enter description..."
+            ></textarea>
+
+            {errors.title && errors.title.type === "required" && (
+              <span className="text-danger ">title is required</span>
+            )}
+          </div>
+          <div className="row">
+            <div className="col-md-6">
+                <select
+                {...register("employeeId", { required: true, valueAsNumber: true })}
+                aria-label="Default select example"
+                type="number"
+                className="form-select"
+              >
+                <option value="" className="text-muted">
+                  User
+                </option>
+                <option >name</option>
+                {/* {
+                    tagsList?.map((tag) =>(
+                      <>
+                      <option key={tag.id} value={tag.id}>{tag.name}</option>
+                      </>
+                    ))
+                  } */}
+              </select>
+                {errors.employeeId && errors.employeeId.type === "required" && (
+                  <span className="text-danger ">No User Selected</span>
+                )}
+            </div>
+            <div className="col-md-6 ">
+                  <select
+                  {...register("projectId", { required: true, valueAsNumber: true })}
+                  aria-label="Default select example"
+                  type="number"
+                  className="form-select mt-sm-3 mt-md-0"
+                >
+                  <option value="" className="text-muted">
```

Comment:
you should pull the projects list here and display them, to be able to test the UI before merging